README.md

```diff
@@ -396,6 +397,50 @@ const balances = await io.getBalances({
 
 </details>
 
+#### `getVaults({ cursor, limit, sortBy, sortOrder })`
+
+Retrieves the vaults of the IO process, paginated and sorted by the specified criteria. The `cursor` used for pagination is the last wallet address from the previous request.
```

Comment:
Consider: `Retrieves all locked-balance user vaults of the IO process...` to help disambiguate from delegation withdraw vaults.

Reply:
f2ec83c